model.py

```diff
@@ -295,7 +293,9 @@ def test_step(self, batch, batch_idx):
 
   def configure_optimizers(self):
     # Train with a lower LR on the output layer
-    LR = 1.5e-3
+    LR = 8.75e-4
+    gamma=0.992
```

Comment:
note that this gamma assignment has no effect, the argument to the function further down is not changed.